garrysmod/gamemodes/sandbox/entities/weapons/gmod_tool/stools/axis.lua

```diff
@@ -58,13 +58,14 @@ function TOOL:LeftClick( trace )
 
 		local Ent1, Ent2 = self:GetEnt( 1 ), self:GetEnt( 2 )
 		local Bone1, Bone2 = self:GetBone( 1 ), self:GetBone( 2 )
-		local Norm1, Norm2 = self:GetNormal( 1 ), self:GetNormal( 2 )
+		--local Norm1, Norm2 = self:GetNormal( 1 ), self:GetNormal( 2 )
+		local _, Norm2 = self:GetNormal( 1 ), self:GetNormal( 2 )

```

Comment:
```suggestion
		local Norm1, Norm2 = self:GetNormal( 1 ), self:GetNormal( 2 )
```
You did this in the wrong place, you probably wanted to do this in TOOL:RightClick